roaring/roaring.go

```diff
@@ -395,32 +395,300 @@ func (b *Bitmap) Intersect(other *Bitmap) *Bitmap {
 	return output
 }
 
-// Union returns the bitwise union of b and other.
-func (b *Bitmap) Union(other *Bitmap) *Bitmap {
+// Union returns the bitwise union of b and other as a new bitmap.
```

Comment:
should read "... of b and others as ..."

Reply:
fixed